src/lib/broadcast.jl

```diff
@@ -187,4 +187,8 @@ end
     sum(xs, dims = dims), Δ -> (placeholder .= Δ,)
   end
 
+  @adjoint function Base.convert(::Type{T}, xs::Array{K,N})  where {T<:CuArrays.CuArray, K, N}
```

Comment:
The `K, N` could be removed here, right? Otherwise LGTM.

Reply:
Should be fixed now, apologies for the delay